ansible/roles/ocp4-workload-ccnrd-stable/tasks/post_workload.yml

```diff
@@ -1,78 +1,72 @@
-    - "user.info: "
-
-
+  agnosticd_user_info:
+    msg: |
+      Individual module guide URLs (in case you need them for direct linking):
+    
 - name: output workshop info guides
   when: ("m1" in modules)
-  debug:
+  agnosticd_user_info:
     msg:
-    - "user.info: Module 1 http://guides-m1-labs-infra.{{ route_subdomain }}"
+      Module 1 http://guides-m1-labs-infra.{{ route_subdomain }}
 
 - name: output workshop info guides
   when: ("m2" in modules)
-  debug:
+  agnosticd_user_info:
     msg:
-    - "user.info: Module 2 http://guides-m2-labs-infra.{{ route_subdomain }}"
+      Module 2 http://guides-m2-labs-infra.{{ route_subdomain }}
 
 - name: output workshop info guides
   when: ("m3" in modules)
-  debug:
+  agnosticd_user_info:
     msg:
-    - "user.info: Module 3 http://guides-m3-labs-infra.{{ route_subdomain }}"
+      Module 3 http://guides-m3-labs-infra.{{ route_subdomain }}
 
 - name: output workshop info guides
   when: ("m4" in modules)
-  debug:
+  agnosticd_user_info:
     msg:
-    - "user.info: Module 4 http://guides-m4-labs-infra.{{ route_subdomain }}"
+      Module 4 http://guides-m4-labs-infra.{{ route_subdomain }}
 
 - name: output workshop info guides
-  debug:
-    msg:
-    - "user.info: "
-    - "user.info: OpenShift credentials for attendees: {{ workshop_openshift_user_name }} / {{ workshop_openshift_user_password }}"
-    - "user.info: CodeReady Workspaces credentials for attendees: {{ workshop_che_user_name }} / {{ workshop_che_user_password }}"
-    - "user.info: "
-    - "user.info: OpenShift Console: {{ console_url }}"
-    - "user.info: Cluster admin login with '{{ ocp_username }}' / 'r3dh4t1!'"
-    - "user.info: "
-    - "user.info: CodeReady Console: https://codeready-labs-infra.{{ route_subdomain }}"
-    - "user.info: Admin login with 'admin' / 'admin'"
-    - "user.info: "
-    - "user.info: NOTE: Workspaces in CodeReady AND service mesh are provisioned asynchronously and may not"
-    - "user.info: be accessible until rollout finishes shortly."
+  agnosticd_user_info:
+    msg: |
+      OpenShift credentials for attendees: {{ workshop_openshift_user_name }} / {{ workshop_openshift_user_password }}
+      CodeReady Workspaces credentials for attendees: {{ workshop_che_user_name }} / {{ workshop_che_user_password }}
+    
+      OpenShift Console: {{ console_url }}
+      Cluster admin login with '{{ ocp_username }}' / 'r3dh4t1!'
+    
+      CodeReady Console: https://codeready-labs-infra.{{ route_subdomain }}
+      Admin login with 'admin' / 'admin'
+    
+      NOTE: Workspaces in CodeReady AND service mesh are provisioned asynchronously and may not be accessible until rollout finishes shortly.
   when: not silent|bool
 
 # Leave this as the last task in the playbook.
 - name: post_workload tasks complete
-  debug:
-    msg: "Post-Workload Tasks completed successfully."
+  agnosticd_user_info:
+    msg: 
+      Post-Workload Tasks completed successfully.
```

Comment:
Did you mean to use a yaml block quote here for the `msg:` line?

Reply:
Updated with a pipe